packages/expo-video/android/src/main/java/expo/modules/video/FullscreenActivity.kt

```diff
@@ -0,0 +1,74 @@
+package expo.modules.video
+
+import android.app.Activity
+import android.os.Bundle
+import android.view.View
+import android.view.WindowInsets
+import android.view.WindowInsetsController
+import android.widget.ImageButton
+import androidx.media3.ui.PlayerView
+
+@androidx.annotation.OptIn(androidx.media3.common.util.UnstableApi::class)
+class FullscreenActivity : Activity() {
```

Comment:
```suggestion
class FullscreenPlayerActivity : Activity() {
```
I think we should rename it to something more descriptive.